src/logic/create.ts

```diff
@@ -16,43 +17,35 @@ import * as providerTemplate from '../templates/provider';
  */
 export async function createProfile(
   basePath: string,
-  id: {
-    scope?: string;
-    name: string;
-    version: DocumentVersion;
-  },
+  profile: ProfileId,
+  version: DocumentVersion,
   usecaseNames: string[],
   superJson?: SuperJson,
+  fileName?: string,
   options?: {
     force?: boolean;
     logCb?: LogCallback;
   }
 ): Promise<void> {
-  let profileName = id.name;
-  let filePath = `${profileName}${EXTENSIONS.profile.source}`;
-  const version = composeVersion(id.version);
-
-  if (id.scope !== undefined) {
-    profileName = `${id.scope}/${profileName}`;
-    filePath = joinPath(id.scope, filePath);
-  }
+  let filePath = `${fileName || profile.id}${EXTENSIONS.profile.source}`;
```

Comment:
Don't know if it right or not, but I didn't expect it will append extension if the extension is present.
Best would be check if extension is present and valid.
Good enough not to append it at all.

```
❯ yarn superface create --profileId communication/validate-email --profile --profileFileName capabilities/communication/validate-email/profile.supr
yarn run v1.22.10
$ /Users/freaz/Workspaces/Superface/station/node_modules/.bin/superface create --profileId communication/validate-email --profile --profileFileName capabilities/communication/validate-email/profile.supr
-> Created capabilities/communication/validate-email/profile.supr.supr (name = "communication/validate-email", version = "1.0.0")
$ echo '<updated super.json>' > 'superface/super.json'
✨  Done in 1.45s.

❯ ls capabilities/communication/validate-email/
profile.supr.supr
```